addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -108,8 +108,8 @@
             </xpath>
             <xpath expr="//page[@name='settings']//field[@name='allow_billable']" position="after">
                 <div invisible="not allow_billable or not allow_timesheets" class="text-muted">
-                    Timesheets without a sales order item are
-                    <field name="billing_type" nolabel="1"/>
+                    Timesheets without a sales order item are reported as
+                    <field name="billing_type" nolabel="1" class="w-25"/>
```

Comment:
I don't think it is needed. 🤷🏼‍♂️ 
```suggestion
                    <field name="billing_type" nolabel="1"/>
```

Reply:
> I don't think it is needed. 🤷🏼‍♂️

I have replied in the task chatter with an attachment. Thank you.